filters/filters.txt

```diff
@@ -1,7 +1,7 @@
-! Version: 6.8.2
+! Version: 6.9.2
```

Comment:
Incorrect version number, should be 6.9.0.

Reply:
```suggestion
! Version: 6.9.0
```

Reply:
im gonna have to open the pr again.

Reply:
nvm ill just use git